docs/tutorials/intro_tutorial.rst

```diff
@@ -879,273 +895,204 @@ True indefinitely.
                 x = self.random.randrange(self.grid.width)
                 y = self.random.randrange(self.grid.height)
                 self.grid.place_agent(a, (x, y))
-
+    
             self.datacollector = DataCollector(
-                model_reporters={"Gini": compute_gini},
-                agent_reporters={"Wealth": "wealth"})
-
+                model_reporters={"Gini": compute_gini}, agent_reporters={"Wealth": "wealth"}
+            )
+    
         def step(self):
             self.datacollector.collect(self)
             self.schedule.step()
 
-We instantiate a BatchRunner with a model class to run, and two
-dictionaries: one of the fixed parameters (mapping model arguments to
-values) and one of varying parameters (mapping each parameter name to a
-sequence of values for it to take). The BatchRunner also takes an
-argument for how many model instantiations to create and run at each
-combination of parameter values, and how many steps to run each
-instantiation for. Finally, like the DataCollector, it takes
-dictionaries of model- and agent-level reporters to collect. Unlike the
-DataCollector, it won't collect the data every step of the model, but
-only at the end of each run.
+We call a batch run with a model class to run, and a dictionary with the
+model parameters where each of the values can either be fixed or an
+iterable. Further, we can specify how many model instantiations to
+create and run at each combination of parameter values, and how many
+steps to run each instantiation for. We can also set the number of
+processes to be run at the same time (default ``None`` uses all
+available processors) as well as the length of the interval after which
+the model and agent reporters of the model class collect data.
 
 In the following example, we hold the height and width fixed, and vary
-the number of agents. We tell the BatchRunner to run 5 instantiations of
-the model with each number of agents, and to run each for 100 steps.\*
+the number of agents. We tell the batch runner to run 5 instantiations
+of the model with each number of agents, and to run each for 100 steps.
 
-We have it collect the final Gini coefficient value.
+We want to keep track of 1. the Gini coefficient value and 2. the
+individual agent’s wealth development.
 
-Now, we can set up and run the BatchRunner:
+Since for the latter changes at each time step might be interesting, we
+set ``i_steps=1``.
```

Comment:
Seeing this in the tutorial I concur with @rht ```i_steps``` is cryptic if this can be changed to ```data_collection_period``` that would be great

Reply:
Should I open a new PR for this or just add these changes here?

Reply:
I think the `i_steps` -> `data_collection_period` rename should be done in a separate PR. 2 smaller PRs are easier to review, and faster to be merged than 1 big PR altogether.

Reply:
Just opened a new PR, will rebase and update this one once the new one is merged.